Automattic/wp-calypso
# Theme Showcase: Remove unused state for opening/closing the showcase

#### Changes proposed in this Pull Request

* Following up on #54361, remove unused state reducers/actions and their related tests. No visuals.

#### Testing instructions

* Switch to this PR, navigate to `/themes`
* Make sure everything still works as intended and that nothing breaks as a result of removing these actions/reducers.
